### ISSUE TITLE
BUG: Falls tab of the memory leak

### ISSUE THREAD

**Issue by helloris25** [AnyChart/AnyChart#1]
Falls tab if between the values of the scale Y of the difference y 0.00001
Падает вкладка, если между значениями в шкале Y разница 0,00001

```
<!doctype html>
<html>
  <head>
    <script src="https://cdn.anychart.com/js/7.13.0/anychart-bundle.min.js"></script>
    <link rel="stylesheet" href="https://cdn.anychart.com/css/7.13.0/anychart-ui.min.css" />
    <style>
      html, body, #container {
        width: 100%;
        height: 100%;
        margin: 0;
        padding: 0;
      }
    </style>
  </head>
  <body>
    <div id="container"></div>
    <script>

anychart.onDocumentReady(function() {
    chart = anychart.line([1.10001, 1.10002]);

    chart.container("container");
    chart.draw();
});
    
    </script>
  </body>
</html>
```

**Comment by loki20beta** [comment]
We expect this bug to be fixed in 7.13.1 release in a couple of weeks.

**Comment by chernetsky** [comment]
Dev preview with the fix is available at http://static.anychart.com/js/DVF-2845-ticks-calculation-fix/anychart-bundle.min.js

**Comment by loki20beta** [comment]
The bug is fixed in 7.13.1

**Comment by helloris25** [comment]
thanks
